docker-compose.yml

```diff
@@ -16,6 +17,7 @@ services:
       - elk
 
   logstash:
+    image: dockerelk_logstash:latest
```

Comment:
I don't like the fact that we need to rely on `docker-compose` first to build the images. `docker-compose` and `docker stack` are two different tools with different purposes and there should not be any dependency on one or the other.